scripts/examples/typescript/README.md

```diff
@@ -0,0 +1,33 @@
+## Typescript for XenAPI Usage
+
+### Examples
+Install lib from npm repository
+```
+$ npm install xen-api-ts
```

Comment:
Is it standard to add -ts to the library name? The end result should be a javascript library along the typing metadata so it doesn't feel right

Reply:
No, there is no standard mentioned this library naming convertion. 
According to the current tsconfig.json and package.json, we can build and publish the TypeScript based NPM package that interops seamlessly with TypeScript and JavaScript, so the -ts is not mandatory.

BTW, there is a official guide to publish npm packages under the [@types](https://www.npmjs.com/~types) organization https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#publish-to-types

Reply:
the other thing is the name `xen-api` was occupied on npmjs, so I have to add -ts suffix for difference.